OwenConferenciasApp/screens/ProfileUserLayout.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {View, Image, TouchableOpacity, Text, StyleSheet, Alert} from 'react-native';
```

Comment:
I think it is a bad practice to call components and do not used them, I suggest to do not call Alert if you are not using it when rendering.

Reply:
Exactly, because is a component that use time of load and that is equals to more app loading time

Reply:
Ok let me erase that component